.circleci/config.yml

```diff
@@ -99,7 +99,7 @@ workflows:
       - test:
          name: "code-check"
          image: "python:3.6"
-         toxenv: "protocheck,generatecheck,mypy,mypy-report,black,flake8,docstrings,verify-sweeps"
+         toxenv: "protocheck,generatecheck,mypy,mypy-report,black,flake8,pyupgrade,docstrings,verify-sweeps"
```

Comment:
This isnt what we want right?
This needs to be something like: pyupgradecheck
otherwise there is no restriction, code will be "upgraded" in an ephemeral directory in circleci and then thrown away.

Reply:
yea, I've verified that it behaves as intended:
<img width="684" alt="image" src="https://user-images.githubusercontent.com/7557205/162514600-4a4c0d0f-5c6d-4c91-8a1f-cf4fe650f4a2.png">